### ISSUE TITLE
Can't produce message using UI

### ISSUE THREAD

**Issue by barmaxence** [provectus/kafka-ui#4068]
### Issue submitter TODO list

- [X] I've looked up my issue in [FAQ](https://docs.kafka-ui.provectus.io/faq/common-problems)
- [X] I've searched for an already existing issues [here](https://github.com/provectus/kafka-ui/issues)
- [X] I've tried running `master`-labeled docker image and the issue still persists there
- [X] I'm running a supported version of the application which is listed [here](https://github.com/provectus/kafka-ui/blob/master/SECURITY.md)

### Describe the bug (actual behavior)

When I try to produce a JSON message through Kafka UI, using Schema Registry, I get an error message

> Content no schema with key or ref "http://json-schema.org/draft-04/schema#"

### Expected behavior

Can produce message

### Your installation details

Kafka stack : 
- confluentinc/cp-kafka
- confluentinc/cp-schema-registry
- provectuslab/kafka-ui (master)

```
version: '2'
services:

  kafka0:
    image: confluentinc/cp-kafka:latest
    container_name: kafka0
    hostname: kafka0
    ports:
      - '9092:9092'
    environment:
      KAFKA_NODE_ID: 1
      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT'
      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT://kafka0:29092,PLAINTEXT_HOST://localhost:9092'
      KAFKA_PROCESS_ROLES: 'broker,controller'
      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
      KAFKA_CONTROLLER_QUORUM_VOTERS: '1@kafka0:29093'
      KAFKA_LISTENERS: 'PLAINTEXT://kafka0:29092,CONTROLLER://kafka0:29093,PLAINTEXT_HOST://0.0.0.0:9092'
      KAFKA_INTER_BROKER_LISTENER_NAME: 'PLAINTEXT'
      KAFKA_CONTROLLER_LISTENER_NAMES: 'CONTROLLER'
      KAFKA_LOG_DIRS: '/tmp/kraft-combined-logs'
      CLUSTER_ID: 'MkU3OEVBNTcwNTJENDM2Qk'

  schemaregistry0:
    image: confluentinc/cp-schema-registry:latest
    container_name: schemaregistry0
    ports:
      - '8085:8085'
    depends_on:
      - kafka0
    environment:
      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'PLAINTEXT://kafka0:29092'
      SCHEMA_REGISTRY_KAFKASTORE_SECURITY_PROTOCOL: 'PLAINTEXT'
      SCHEMA_REGISTRY_HOST_NAME: 'schemaregistry0'
      SCHEMA_REGISTRY_LISTENERS: 'http://schemaregistry0:8085'
      SCHEMA_REGISTRY_SCHEMA_REGISTRY_INTER_INSTANCE_PROTOCOL: 'http'
      SCHEMA_REGISTRY_LOG4J_ROOT_LOGLEVEL: 'INFO'
      SCHEMA_REGISTRY_KAFKASTORE_TOPIC: '_schemas'

  kafka-ui:
    image: provectuslabs/kafka-ui:master
    container_name: kafka-ui
    ports:
      - '8080:8080'
    depends_on:
      - kafka0
      - schemaregistry0
    environment:
      KAFKA_CLUSTERS_0_NAME: 'local'
      KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: 'kafka0:29092'
      KAFKA_CLUSTERS_0_SCHEMAREGISTRY: 'http://schemaregistry0:8085'
      DYNAMIC_CONFIG_ENABLED: 'true'
```

.NET producer and consumer

```
<PackageReference Include="Confluent.Kafka" Version="2.2.0" />
<PackageReference Include="Confluent.SchemaRegistry" Version="2.2.0" />
<PackageReference Include="Confluent.SchemaRegistry.Serdes.Json" Version="2.2.0" />
```

```
using var producer = new ProducerBuilder<string, TValue>(producerConfig)
            .SetKeySerializer(Serializers.Utf8) // same issue with or without this line
            .SetValueSerializer(new JsonSerializer<TValue>(schemaRegistry))
            .Build();
```

### Steps to reproduce

- Produce a message on topic `X`, with the configuration above
- Producer auto create subject `X-value` in schema registry
- Message is produced
- On Kafka UI, create a message on topic 'X'

### Screenshots

<img width="1792" alt="Capture d’écran 2023-07-28 à 12 12 48" src="https://github.com/provectus/kafka-ui/assets/15143763/1ce48ad8-dfc1-4d61-9b37-29ff5ab9cfdf">
<img width="1784" alt="Capture d’écran 2023-07-28 à 12 15 42" src="https://github.com/provectus/kafka-ui/assets/15143763/072522cb-a5e6-4bd7-ad78-02f3f68d4c44">
<img width="1774" alt="Capture d’écran 2023-07-28 à 12 16 17" src="https://github.com/provectus/kafka-ui/assets/15143763/8ddb133c-82be-4b67-963b-c93689b8cbb6">

### Additional context

To sum up, no problem to produce/consume messages with .NET Kafka package, just an issue to create a message with Kafka UI.

Thx for your help

**Comment by Haarolean** [comment]
Hi, it's a duplicate of #2354. Please follow the original issue.
